projects/manager/src/nop.spec.ts

```diff
@@ -0,0 +1,7 @@
+// this file can be removed as soon as there is a real test in the project
+
+describe('NopTest', () => {
+    it('should work if true is returned', () => {
+        expect(true).toBeTruthy();
+    });
+});
```

Comment:
I only added this test here, because the linter wants to have at least one test. This exact test is present in all dashboards so I thought this was the way to do it. Maybe we could change the linter but I don't know how.